content/pt-br/docs/reference/glossary/wg.md

```diff
@@ -0,0 +1,19 @@
+---
+title: WG (Grupo de Trabalho)
+id: wg
+date: 2018-04-12
```

Comment:
I believe this date needs to be updated with the creation date of this translated file. @jcjesus is that correct?

Reply:
Hi @stormqueen1990, I´m not sure, but if this file it´s a new one, could be updated to the date creation file.

Reply:
Hi @jcjesus and @stormqueen1990, in spanish and chinese have not changed this date. They kept the original date.

Reply:
Ok, @MrErlison, don´t worries, I´ll approve now!

Reply:
I'm not worried. I'm more interested in what the pattern. Thank you for your attention. I'm new here, in Github.

Reply:
I understood you @MrErlison, but this was the first time questioned the file´s date. That you saw in other translate teams, and this is could able to answer us, how they made before is the pattern.